docs/quick-start.md

```diff
@@ -22,6 +22,13 @@ kubectl apply -n numaflow-system -f https://raw.githubusercontent.com/numaproj/n
 kubectl apply -f https://raw.githubusercontent.com/numaproj/numaflow/stable/examples/0-isbsvc-jetstream.yaml
 ```
 
+Port forward the UI to https://localhost:8443/
```

Comment:
This is not part of installation, should not be here.

Reply:
How about adding a ## Port Forwarding section?

Reply:
The expected user of Numaflow is someone who knows Kubernetes, and this is a very common operation in k8s world, don't think this needs to be in a section.

Reply:
Makes sense. I moved `kubectl -n numaflow-system port-forward deployment/numaflow-server 8443:8443` command out of `A Simple Pipeline` section because it's shared by both pipelines so it could be executed before any pipeline creation. Do you think I should keep it in the `A Simple Pipeline` then?

Reply:
The doc currently does not have sections for each command, it is a single blob with commands and their outputs. Should we split it out to make it more readable?

Reply:
I made these changes assuming readers are new to k8s like myself. Since numaflow is targeting users who knows k8s, I agree that we can keep this quickstart doc as it is. I will update this PR to just fix typo.